README.md

```diff
@@ -135,17 +136,27 @@ npm run build
 
 ## Developing
 
-For the [mystmd](https://github.com/executablebooks/mystmd) library on GitHub, `git clone` and you can install the dependencies and then create a local copy of the library with the `npm run dev` command.
+The `mystmd` libraries and command line tools are written in TypeScript, and require [NodeJS and npm](https://nodejs.org) for local development. The `mystmd-py` package, which is a thin Python wrapper around the `mystmd` bundle, can be installed by users using `pip` or `conda`. If you have already installed `mystmd` (e.g. via `pip` or `conda`), it is recommended that you uninstall it (or deactivate the relevant environment) before using the local development instructions below.
+
+For local development, [clone the repository](https://github.com/executablebooks/mystmd) and install the dependencies using npm. You can then build the libraries (`npm run build`) and then optionally link to your globally installed `mystmd` in node using the `npm run link` command.
 
 ```shell
 git clone git@github.com:executablebooks/mystmd.git
 cd mystmd
 npm install
 npm run build
-npm run dev
+npm run link
```

Comment:
When I run `npm run link` I got the following error:

```shell
❯ npm -v
10.1.0
❯ node -v
v20.9.0
❯ npm run link
npm ERR! Missing script: "link"
npm ERR! 
npm ERR! Did you mean this?
npm ERR!     npm run lint # run the "lint" package script
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in: /Users/choldgraf/.npm/_logs/2024-03-20T16_31_12_437Z-debug-0.log
```

Could you advise?

Reply:
```suggestion
npm link
```